server.js

```diff
@@ -258,7 +262,8 @@ function dgUnlockForClient (clientId) {
     return {status: false, unlocked: null}
 }
 
-
+var connectedClient;
```

Comment:
We have to maintain an array of client here to send the dbConnectivity state to all of them periodically.